.github/workflows/publish.yml

```
@@ -31,14 +31,33 @@
 
       - name: Build the project
         run: dotnet build ${{ env.PROJECT_PATH }} --configuration Release --no-restore
 
-      - name: Pack the NuGet package
-        run: dotnet pack ${{ env.PROJECT_PATH }} --configuration Release --no-build --output ./nupkg -p:Version=${{ steps.version.outputs.version }}
+      # Pack: PowerShell で安全に多行文字列を渡す
+      - name: Pack the NuGet package (safe)
+        shell: pwsh
+        run: |
+          $notes = @'
+      ${{ steps.notes.outputs.notes }}
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 53: could not find expected ':'

(syntax-check)